airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/record_buffer/SerializedBufferingStrategy.kt

```diff
@@ -116,6 +115,12 @@ class SerializedBufferingStrategy
         LOGGER.info { "Flushing completed for ${stream.name}" }
     }
 
+    @Throws(Exception::class)
+    override fun flushSingleStream(stream: AirbyteStreamNameNamespacePair) {
+        val buffer = allBuffers[stream]!!
```

Comment:
nit: allBuffers.getValue

Reply:
Why?

Reply:
I thought it was a kotlin convention to use the `map.getValue(x)` instead of `map[x]!!`, but I couldn't back that up with any actual data.
And then I realized that there is a difference between `map.getValue(x)` and `map[x]!!`: in kotlin, a map can be defined with a `default` value that would be returned by `getValue(x)` if `x` is not a key of the map, while `[x]` will return `null` regardless of the default.
In this case, I don't think it matters, so go with your heart